2-mandatory/6-journey-planner.js

```diff
@@ -92,7 +97,10 @@ function isAccessibleByTransportMode() {}
    - Returns the name of the location
       e.g: "Tower Bridge"
 */
-function getLocationName() {}
+function getLocationName(locationAndTransports) {
+  let locationAndTransportsCopy = [...locationAndTransports];
```

Comment:
Hm... so we create the whole new array to return the first element. Why don't we

```javascript
return locationAndTransportModes[0];
```